### ISSUE TITLE
ArcFace loss init error

### ISSUE THREAD

**Issue by tkuanlun350** [KevinMusgrave/pytorch-metric-learning#177]
Hi ! Thanks for this awesome library. I have met an error when I try to use arcface loss:
`    
loss_func = losses.ArcFaceLoss(11, 512, margin=28.6, scale=64, **kwargs)
`
It gives me

> Traceback (most recent call last):
  File "train_img.py", line 152, in <module>
    main(config)
  File "train_img.py", line 80, in main
    loss_func = losses.ArcFaceLoss(11, 512, margin=28.6, scale=64, **kwargs)
TypeError: __init__() got multiple values for argument 'scale'

I am using the latest release
Did I missing something ?

**Comment by KevinMusgrave** [comment]
Hmm, this worked for me:
```python
from pytorch_metric_learning import losses
loss_func = losses.ArcFaceLoss(11, 512, margin=28.6, scale=64)
print(loss_func)
embeddings, labels = torch.randn(32,512), torch.randint(0,10,(32,))
print(loss_func(embeddings, labels))
```
To clarify, the ```**kwargs``` shown in the documentation was meant to signify that there are other keyword arguments that are inherited from the base class, such as ```reducer``` and ```distance```.

Let me know if the above snippet works

**Comment by tkuanlun350** [comment]
yeah I just pass empty dict as kwargs to simply things. It should not relate to the issue :)
I tried the code snippet and still shows the error. Maybe something relate to my environment setup since there is no duplicate use of scale in the source code (I first install with conda then pip install to the latest version.)

Let me setup a clean environment and get back to you. Thank you !

BTW
`
pytorch_metric_learning.__version__ = 0.9.90.dev0
`

**Comment by KevinMusgrave** [comment]
If it's possible for you to use pytorch 1.6, then you should upgrade pytorch-metric-learning to v0.9.90, which is newer than your version. I think that bug may have existed in the pre-release you're using.

**Comment by tkuanlun350** [comment]
It fixed after I update the pytorch to 1.6 and v0.9.90. Thanks 👍
